src/reducers/fileTree.spec.tsx

```diff
@@ -764,7 +764,7 @@ describe(__filename, () => {
             messageMap,
             position: RelativePathPosition.previous,
           }),
-        ).toEqual({ path: file1, uid: global1 });
+        ).toEqual({ line: null, path: file1, uid: global1 });
```

Comment:
Same again. It would be good to verify that all messages in the setup data have unique uids, though. I won't comment on all of them but I did a quick glance and I think they should all use `expect.objectContaining()`.